client/client_test.go

```diff
@@ -2732,6 +2790,14 @@ func TestRotateKeyAfterPublishServerManagementChange(t *testing.T) {
 		data.CanonicalTargetsRole:  false,
 		data.CanonicalRootRole:     false,
 	})
+	// check that the snapshot remote rotation creates new keys
```

Comment:
Ah ok - each invocation of `testRotateKeySuccess` wipes the repo and starts a new server - so if we wanted to test the additional rotation you'd probably want to do :

`testRotateKeySuccess(t, true, map[string]bool{data.CanonialSnapshotRole: true})`

That first boolean says whether to start the the repo off with a server-managed snapshot key.